tests/test-rest-api.php

```diff
@@ -539,4 +480,62 @@ public function test_post_with_featured_image() {
 			(int) get_post_thumbnail_id( $post_id )
 		);
 	}
+
+	public function test_set_modified_date() {
+		$modified = '2001-02-03 04:05:06';
+
+		$data = $this->create_post_request_with_defaults( compact( 'modified' ) );
+
+		$this->assertFalse( empty( $data['posts'][0]['post_id'] ) );
+		$post_id = $data['posts'][0]['post_id'];
+
+		// Confirm that the image was stored as the post thumbnail.
```

Comment:
Looks like this comment was copied from another test and isn't applicable here.
```suggestion
		// Confirm that the post modified time was set correctly.
```